terraform/spec/dependabot/terraform/file_parser_spec.rb

```diff
@@ -44,10 +73,10 @@
       specify { expect(subject.length).to eq(5) }
       specify { expect(subject).to all(be_a(Dependabot::Dependency)) }
 
-      it "has the right details for the first dependency (default registry with version)" do
-        expect(subject[0].name).to eq("hashicorp/consul/aws")
-        expect(subject[0].version).to eq("0.1.0")
-        expect(subject[0].requirements).to eq([{
+      it "has the right details for the dependency (default registry with version)" do
+        expect(subject[2].name).to eq("hashicorp/consul/aws")
+        expect(subject[2].version).to eq("0.1.0")
+        expect(subject[2].requirements).to eq([{
```

Comment:
It took me a minute to get the diff, but we are now sorting the returned dep list by name so it's more obviously deterministic, good call.